tests/Makefile

```diff
@@ -0,0 +1,29 @@
+all: lint unit_tests
+
+lint:
```

Comment:
@antoviaque I finally got it working, but had to overcome a mental block.  I tried to answer the question why prospector does not work here, while it works fine for the instance manager.  The answer turned out to be that it currently isn't working for the instance manager as well, since the current version of prospector is incompatible with pylint 1.6.  Due to the opacity of prospector, we haven't noticed this when upgrading pylint to version 1.6.  I reverted the pylint version to 1.5.6 here, and now everything is working fine.  We need to fix this in the instance manager as well, but it is likely that we already have added a few pylint violations since it stopped working.  I'll take a look and create a ticket to fix this if appropriate.  (It could also be useful to get rid of prospector altogether, but I'll elaborate on the new ticket.)

Regarding the django plugin, another solution would be to remove it from the pylint configuration file and load it via a command-line flag to prospector in the instance manager.  This way, the confiugration files in both repos could be identical, and only the command-line invocations slightly different.  This seems a cleaner solution to me then trying to fix the configuration file with sed.

We definitely didn't plan for all this during the discovery, since at that time I didn't expect to implement the certificate management on the load balancer, and that's the only reason I added Python code at all.  Everything else is more easily done in shell script.  I don't expect the code of the load balancer to signifcantly grow in the future, by the way, although code has a tendency to do so even if you don't expect it.